package.json

```diff
@@ -14,8 +14,8 @@
   "license": "MIT",
```

Comment:
Update to `3.0.0` version

Reply:
Do we have a breaking change here?

Reply:
Now, we don't fallback to `document.querySelectorAll` if page does not contan Vue.

https://github.com/DevExpress/testcafe-vue-selectors/pull/32/files#diff-1fdf421c05c1140f6d71444ea2b27638L73

Reply:
I doubt whether it deserves to be called as a breaking change since our plugin is created for VueJS applications? /cc @kirovboris @VasilyStrelyaev 


> MAJOR version when you make incompatible API changes,

Reply:
I suppose `2.1.0`

Reply:
Ok. Let it be `2.1.0`